.github/.jazzy.yaml

```
@@ -11,31 +11,34 @@ author: WWT and Tyler Thompson
 documentation: .github/guides/*.md
 abstract: .github/abstract/*.md
 custom_categories:
-- name: How to use SwiftCurrent with SwiftUI
-  children:
-    - Getting Started with SwiftUI
-    - Working with NavigationView
-- name: How to use SwiftCurrent with UIKit
+- name: Overview
   children:
-    - Storyboards Intro
-    - Programmatic UIKit Intro
+    - Why This Library
+    - Installation
 - name: Creating Workflows
   children:
     - FlowRepresentable
-    - PassthroughFlowRepresentable
     - Workflow
+    - PassthroughFlowRepresentable
+- name: How to use SwiftCurrent with SwiftUI
+  children:
+    - Getting Started with SwiftUI
+    - Working with NavigationView
 - name: Creating Workflows in SwiftUI
   children:
+    - WorkflowLauncher
     - View
-    - ViewControllerWrapper
     - WorkflowItem
-    - WorkflowLauncher
+- name: How to use SwiftCurrent with UIKit
+  children:
+    - Getting Started with Programmatic Views
```

This is getting cut off. It says "Getting Started with Programmatic V". That's less than ideal, are we just accepting that for now?

---

I'm inclined to accept that issue for this PR and have a follow-up PR to hash out Abstracts and names.  But I'm also open to another rename.  I would want it to read well for all 3 Getting Started.  I played with:
```suggestion
    - Getting Started Programmatically
```
But that didn't read as well in this instance, nor did it translate to the other 2.

We can start a thread here for other names we could use.

---

The issue is that these have to be globally unique, or at least I could not see anything from Jazzy that let you map file names to headers. So that tends to make us want to be more verbose.

I really wish our docs didn't cut off. Have you considered:
```suggestion
- name: How to use SwiftCurrent with UIKit
  children:
    - With Programmatic Views
    - With Storyboards
```

---

I like how the UIKit section looks with children

---

I can see that looking pretty good on the sidebar.  I'm not sure for SwiftUI though, let's see:

```
- name: How to use SwiftCurrent with SwiftUI
  children:
    - With SwiftUI
```
Hmm.... What if we did something like:

```
- name: How to use SwiftCurrent with UIKit
  children:
    - Using Programmatic Views
    - Using Storyboards
```

Then SwiftUI might look like this:
```
- name: How to use SwiftCurrent with SwiftUI
  children:
    - Using View Swapping
```

---

I sort of hate "Using View Swapping" but I really like everything else.

---

Is it the calling out that it uses "View Swapping" or that there is terminology better suited for swapping out the views?

Or option 3, you hate it because it's the only in the list?  And if we had:
```
  - Using View Swapping
  - Using NavigationViews
```

Would be better?  Though those would link to wildly different articles...

---

I sort of didn't like View Swapping as a term.

---

Ok so:
* Rename UIKit articles to Using
* Leave SwiftUI the way it is

Then we'll get this PR merged and come back to more naming/Abstracts later?